src/megachatapi_impl.cpp

```diff
@@ -9815,6 +9925,12 @@ const MegaChatContainsMeta* JSonUtils::parseContainsMeta(const char *json, uint8
                 containsMeta->setGeolocation(geolocation);
                 break;
             }
+            case MegaChatContainsMeta::CONTAINS_META_GIPHY:
+            {
+                MegaChatGiphy* giphy = parseGiphy(document);
```

Comment:
Despite the old code doesn't use smart pointers here, maybe it's time to return a `unique_ptr<>` here? so the ownership of the object is easier to track.